.github/workflows/build_and_push.yml

```diff
@@ -44,7 +44,8 @@ jobs:
           docker build \
           --build-arg git_sha=$GITHUB_SHA \
           -t $REGISTRY/${{ matrix.image }}:$GITHUB_SHA \
-          -t $REGISTRY/${{ matrix.image }}:latest .
+          -t $REGISTRY/${{ matrix.image }}:latest . \
+          -f ci/Dockerfile.lambda
```

Comment:
line 42 will put you in the `api` directory as defined on line 26, which doesn't exist. I would recommend removing line 42.